index.html

```diff
@@ -3016,11 +3016,42 @@ <h3 id="applying">
         repository</a> on GitHub.
       </p>
     </section>
-    <section id="internationalization">
+    <section id="internationalization" class="appendix">
       <h2>
-        internationalization
+        Internationalization
       </h2>
-      <div class="issue" data-number="323"></div>
+      <p>
+        It is expected that authors will localize the content of a manifest by
+        using one of the following options:
+      </p>
+      <dl>
+        <dt>
+          Dynamically setting the language:
+        </dt>
+        <dd>
+          This can include, for instance, asking the user what their preferred
+          language is and dynamically adding or replacing the manifest link
+          relationship to the document based on that language preference (e.g.,
+          using a URL like "manifest?lang=fr").
+        </dd>
+        <dt>
+          Using content-negotiation, or geo-ip, etc. on the server:
+        </dt>
+        <dd>
+          The server that hosts the web application could attempt to
+          predetermine the user's language by using a geo-ip look-up or by
+          using content negotiation (e.g., using [[RFC7540]]'s
+          "<code>Accept-Language</code>" header, or even a custom HTTP header).
+        </dd>
+      </dl>
+      <p>
+        Given the options above, developers need to be mindful of the
+        end-user's privacy with respect to their preferred language. Sending
+        the user's preferred language in the clear is generally not ok, as it
+        reveals personal information about an end-user. As such, developers are
+        encouraged to use [[TLS]] to reduce the chances of pervasive monitoring
+        of their Web applications [[RFC7258]].
```

Comment:
I'm going to try to clarify that it's for when the user explicitly provides the language (i.e., not applicable so much for Accept-Languages, as those are usually just pre-configured in the UA).